scout/server/blueprints/cases/templates/cases/phenotype.html

```diff
@@ -200,29 +200,41 @@ <h6 class="mt-3 ms-3"><span class="fa fa-stethoscope"></span>&nbsp;Phenotype gro
       <h6 class="mt-3"><span class="fa fa-stethoscope"></span>&nbsp;Phenotype terms (<a target="_blank" class="" href="{{ url }}" rel="noopener">HPO</a>)</h6>
     </div>
 
-    <!-- Add new HPO terms to case/individuals -->
-    <form method="POST" action="{{ url_for('cases.phenotypes', institute_id=institute._id, case_name=case.display_name) }}">
-      <div class="d-flex justify-content-around">
-        <div>
+    <!-- Add new HPO terms to case/individuals or export phenopacket -->
+  <div class="row mb-3">
+  <form method="POST" action="{{ url_for('cases.phenotypes', institute_id=institute._id, case_name=case.display_name) }}">
+      <div class="row">
+        <div class="col">
           <input name="hpo_term" id="hpo_term" class="typeahead_hpo form-control" data-provide="typeahead" autocomplete="off" required placeholder="Search..." oninput="toggleClickableButtons('assign-phenotype-button', 'hpo_term');">
         </div>
-        <div>
-          <select name="phenotype_inds" multiple class="selectpicker" data-style="btn-secondary">
+        <div class="col">
+          <select name="phenotype_inds" multiple class="selectpicker form-control" data-style="btn-secondary form-control">
             {% for ind in case.individuals %}
               <option value="{{ind.individual_id}}|{{ind.display_name}}" {{"selected" if ind.phenotype==2 }}>
                 {{ind.display_name}}
               </option>
             {% endfor %}
           </select>
         </div>
-        <div class="mr-3">
-          <button class="ms-3 btn no-hover btn-secondary form-control"  id="assign-phenotype-button">Assign Phenotype</button>
+        <div class="col">
+          <button class="btn no-hover btn-secondary form-control" id="assign-phenotype-button">Assign</button>
         </div>
       </div>
     </form>
+    </div>
+    <div class="row mb-3">
+      <label for="import-phenotype-modal-btn" class="col col-form-label">Phenopacket file:</label>
+      <div class="col">
+        <button class="btn btn-secondary form-control" id="import-phenotype-modal-btn" data-bs-toggle="modal" data-bs-target="#import-phenotype">Import</button>
+      </div>
+      <div class="col">
+        <form method="POST" action="{{ url_for('cases.phenotype_export', institute_id=institute._id, case_name=case.display_name) }}">
```

Comment:
Right, that was intended after switching to affected level, but agreed, having export from case level as well makes a lot of sense.

Reply:
Ok, good, this 
![Screenshot 2022-12-14 at 09 11 29](https://user-images.githubusercontent.com/758570/207542766-7f748143-507a-43e8-9d0c-f5a92b1310b4.png)
now gives this
<img width="495" alt="Screenshot 2022-12-14 at 09 11 21" src="https://user-images.githubusercontent.com/758570/207542762-abc0e791-cf6f-4c5e-9d50-ecbea96a41ca.png">
Notice no Curly hair in export, since assigned only to unaffected.